main.py

```diff
@@ -0,0 +1,31 @@
+from fastapi import FastAPI
+from pymongo.cursor import Cursor #tools for iterating in MongoDB
+from bson.objectid import ObjectId
+from app.models import Coffee
+from app.database import coffees_collection
+
+
+app= FastAPI()

```

Comment:
please add title to your app and short summary. Here are some docs about documenting your API: https://fastapi.tiangolo.com/tutorial/metadata/